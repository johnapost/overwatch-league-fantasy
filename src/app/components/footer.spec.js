import React from "react";
import { shallow } from "enzyme";
import Footer from "./footer";

describe("Footer", () => {
  const wrapper = shallow(<Footer />);

  it("should render", () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
