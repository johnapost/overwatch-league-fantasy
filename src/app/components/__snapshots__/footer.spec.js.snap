// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render 1`] = `"<footer class=\\"jsx-1140642571\\"><div class=\\"ant-row\\"><div class=\\"ant-col-sm-6 ant-col-md-4\\"></div><div class=\\"ant-col-sm-12 ant-col-md-16\\"><p class=\\"jsx-1140642571\\">Special thanks to Overwatch and the Overwatch League for existing.</p><p class=\\"jsx-1140642571\\">All trademarks belong to their owners.</p></div></div></footer>"`;
